test/unit/authentication/authentication_test.js

```diff
@@ -71,6 +71,9 @@ describe('external browser authentication', function () {
   const connectionConfig = {
     getBrowserActionTimeout: () => BROWSER_ACTION_TIMEOUT,
     getProxy: () => {},
+    getDisableConsoleLogin: () => {
```

Comment:
nit: can this be simplified to one line `() => true,`

Reply:
Fixed